src/openapi/peridio-admin-openapi.yaml

```diff
@@ -3480,6 +3516,62 @@ components:
       schema:
         type: string
         format: date-time
+    event_inserted_at_lt:
+      name: inserted_at_lt
+      in: query
+      description: |
+        The ending date point of the range of the query
```

Comment:
Add:

> Must be greater than `event_inserted_at_gte`. The duration they specify must be <= 24 hours.

Reply:
Is this field required if `event_inserted_at_gte` i supplied?

Reply:
No, since by default event_inserted_at_gte is `now - 24h` so you can only provide only `inserted_at_lt` as long is within the range of `(now - 24h) <= now`

Reply:
Beautiful